crates/bevy_mesh/src/mesh.rs

```diff
@@ -1167,16 +1167,34 @@ impl Mesh {
                 // If there aren't enough indices to make a triangle, then an empty vector will be
                 // returned.
                 let iterator = match indices {
-                    Indices::U16(vec) => FourIterators::Third(
-                        vec.as_slice()
-                            .windows(3)
-                            .flat_map(move |indices| indices_to_triangle(vertices, indices)),
-                    ),
-                    Indices::U32(vec) => FourIterators::Fourth(
-                        vec.as_slice()
-                            .windows(3)
-                            .flat_map(move |indices| indices_to_triangle(vertices, indices)),
-                    ),
+                    Indices::U16(vec) => {
+                        FourIterators::Third(vec.as_slice().windows(3).enumerate().flat_map(
+                            move |(i, indices)| {
+                                if i % 2 == 0 {
+                                    indices_to_triangle(vertices, indices)
+                                } else {
+                                    indices_to_triangle(
+                                        vertices,
+                                        &[indices[1], indices[0], indices[2]],
```

Comment:
In `bevy_math`, we currently reverse the orientation of `Triangle3d` by swapping the first and last vertices; maybe these should be consistent with that?

Reply:
I followed [documentation for `PrimitiveTopology::TriangleStrip`](https://docs.rs/wgpu-types/latest/wgpu_types/enum.PrimitiveTopology.html#variant.TriangleStrip) which says:

> Vertices 0 1 2 3 4 5 create four triangles 0 1 2, 2 1 3, 2 3 4, and 4 3 5